doc/source/conf.py

```diff
@@ -3,6 +3,10 @@
 # SimPhoNy-Mayavi documentation build configuration file
 #
 import sys
+import os
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__name__), '..', '..')))
```

Comment:
This is for importing `simphony_mayavi` without having installed it (or setup.py develop) beforehand?